jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh

```diff
@@ -457,6 +457,14 @@ function generate_external_datasource_xml() {
                            <background-validation-millis>${millis}</background-validation-millis>"
     fi
 
+
+    # allows for specificaiton of a stale-connection-checker
+    stale_checker=$(find_env "${prefix}_STALE_CONNECTION_CHECKER")
```

Comment:
Would you mind logging a CLOUD_XXX issue for this? 
We have now a dual way of handling env var. sed patching and CLI operations sent to the server.  https://github.com/wildfly/wildfly-cekit-modules/blob/9bdb4feec1c137f93e919d0d7ad572333b6d234e/jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh#L549
So you will need to update this part too.
We are also adding new env var to https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/datasources/module.yaml for documentation purpose.

Reply:
Sorry, not sure where to create a CLOUD_XXX issue... Can you share the link?  First time contributing here :)

Reply:
Sure, you can log a JIRA using https://issues.jboss.org the project to log the issue against is Cloud Enablement. Then you can use the created CLOUD-XXX id in the commit description, for example: "Fix for CLOUD-XXX: add ability to specify a class for a stale connection"

Reply:
ticket has been opened -
https://issues.jboss.org/browse/CLOUD-3427

Cheers!